### DISCUSSION TITLE
How do I get the user information that wrote activity on kimai?

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4789] [Plugins]
How do I get the user information that wrote activity on kimai?

Why is there no user column in kimai2_activity table?
I want to send a notification email to the user who wrote the activity if I didn't write the timesheet for that activity. 
@kevinpapst  How can I know the information of the user who created the activity?

**Comment by kevinpapst** [comment]
Subscribe to https://github.com/kimai/kimai/blob/main/src/Event/ActivityCreatePostEvent.php and then fetch the currently logged in User from the Security context.

**Reply by seleondevel** [reply]
By security context, you mean you can get login user information by using $token->getUser() in the getTokenStorage Interface function, right?
